### PR TITLE
Fix typing for Cryptodomex

### PR DESCRIPTION
**Pull request by tpazderka**
- [ ] Any changes relevant to users are recorded in the `CHANGELOG.md`.
- [ ] The documentation has been updated, if necessary.
- [ ] New code is annotated.
- [ ] Changes are covered by tests.
---

### CODE REVIEW

File: src/oic/utils/aes.py [hunk 1]
```diff
@@ -16,7 +17,11 @@
 
 __author__ = "rolandh"
 
-POSTFIX_MODE = {"cbc": AES.MODE_CBC, "cfb": AES.MODE_CFB, "ecb": AES.MODE_CFB}
+POSTFIX_MODE: dict[str, Union[Literal[2], Literal[3]]] = {
```
Comment (schlenk) [inline src/oic/utils/aes.py:20]: seems it should be typing.Dict[] to work, according to the test output.

**Review by schlenk — CHANGES_REQUESTED** [review]
Looks good besides the small glitches with dict vs. typing.Dict

File: src/oic/utils/aes.py (hunk 1 above)
Comment (tpazderka) [reply src/oic/utils/aes.py]: Yes, that is python 3.9 only :/ Fixed.

**Review by schlenk — APPROVED** [review]
LGTM

**Pull request merged**